src/models/tools/tool-tile.ts

```diff
@@ -61,7 +61,8 @@ export const ToolTileModel = types
       return self.content.type === kPlaceholderToolID;
     },
     get placeholderSectionId() {
-      return (self.content.type === kPlaceholderToolID) ? self.content.sectionId : undefined;
+      return (self.content.type === kPlaceholderToolID) ? 
+        (self.content as PlaceholderContentModelType).sectionId : undefined;
```

Comment:
I have a different implementation of this in my upcoming PR, but we can discuss that when the PR lands.